installer/IF20_cve_search.sh

```diff
@@ -56,7 +56,7 @@ IF20_cve_search() {
 
       #xargs sudo apt-get install -y < requirements.system
       while read -r TOOL_NAME; do
-        apt-get install -y "$TOOL_NAME"
+        apt-get install -y "$TOOL_NAME" --no-install-recommends
```

Comment:
Here it breaks on my Kali setup :( 

<img width="834" alt="Screenshot 2022-09-23 at 22 35 47" src="https://user-images.githubusercontent.com/17053360/192053603-869296f4-88b0-4399-bf12-33c7813dbd9b.png">

Reply:
okey, it is an old Kali version: `Release:        2021.4` will update it and run it again, maybe then it will work.

Reply:
Updating Kali was the solution to this issue.